src/app/pages/HabitList/HabitList.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PlusIcon from '../../components/Icons/PlusIcon';
+import PageLink from '../../components/PageLink/PageLink';
+import SwipeDate from '../../components/SwipeDate/SwipeDate';
+import styles from './HabitList.module.css';
+
+type Habit = {
+  id: string;
+  name: string;
+  color: string;
+  dateCreated: string;
+  datesCompleted: string[];
+  rating: string;
+};
+
+function toListElement(habit: Habit): JSX.Element {
```

Comment:
Actually, this is very similar to a Component (function which returns JSX).
You could convert it:
```suggestion
function ListElement ({ habit }: { habit: Habit }): JSX.Element {
```

If you do so, the `key` should not be set inside the component.

Reply:
I think I will refactor the ListElement component later on, as it's content might still change at this point.